apps/i18n/spritelab/en_us.json

```diff
@@ -193,6 +193,7 @@
   "changeBackground": "The background was not changed.",
   "allSayBlocks": "Make sure each sprite uses a `say` block. \n<xml><block type=\"gamelab_spriteSay\"><title name=\"SPEECH\">Hello, world!</title><value name=\"SPRITE\"><block type=\"gamelab_allSpritesWithAnimation\"><title name=\"ANIMATION\">\"default\"</title></block></value></block></xml>",
   "clickAnySprite": "Make sure to click on any sprite after pressing *\"Run\"*.",
+  "clickAllSprites": "Make sure to click on each sprite after pressing *\"Run\"*.",
```

Comment:
sorry for not being consistent about this before, but we should try to keep this file alphabetized by key. That makes it much easier to find strings and avoid adding duplicates. Can you do that either in this PR or in a follow up?

Reply:
I can do that in a follow-up PR. On that topic, I also wonder if there are other groupings worth considering, such as by lesson or concept. Happy to follow your guidance here.

Reply:
All of the i18n files across the codebase are alphabetized so I think we should do that here. If you want to group by lesson, it could make sense to add a prefix to each key?